src/w3d/renderer/w3d_util.h

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D Utillity class
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#pragma once
+#include "always.h"
+
+struct IOVector3Struct;
+class ShaderClass;
+class Vector4;
+class Vector3;
+struct W3dRGBAStruct;
+struct W3dRGBStruct;
+struct W3dShaderStruct;
+
+class W3dUtilityClass
```

Comment:
If all the members are static and public we are just using this as a namespace so may as well make it `namespace W3dUtilityClass`

Reply:
Its a class in the original and I see no reason not to make it a class in Thyme at this point for consistency.